src/components/cc-global-header.vue

```diff
@@ -108,11 +108,20 @@ export default defineComponent({
       type: String,
       default: "Menu",
     },
+    base_url: {
+      type: String,
+      required: true,
+    },
+    donation_url: {
+      type: String,
+      required: true,
+    },
   },
   data() {
     return {
       isBurgerMenuActive: false,
       menus: {},
+      donation_link: this.donation_url,
```

Comment:
Try accessing the `donation_url` directly in the template, rather than aliasing it with `donation_link`:

https://vuejs.org/v2/guide/components.html#Passing-Data-to-Child-Components-with-Props